struct_conversion/ravenCobraWrapper.m

```diff
@@ -272,6 +272,10 @@
     %Mandatory RAVEN fields
     newModel.rxns=model.rxns;
     newModel.mets=model.mets;
+    if ~isfield(model,'comps')
+        model.comps = setdiff({''},unique(regexprep(model.mets,'.*\[[^\]]+\]$','')));
```

Comment:
This does give the desired result on my PC, what about the following suggestion:

```suggestion
        model.comps = unique(regexprep(model.mets,'.*\[([^\]]+)\]$','$1'));
```

Reply:
You are absolutely right. 
Not sure, what I was thinking there :/

Reply:
:) You should be able to accept the suggested change here on GitHub?